HelpHunterBE/appsettings.json

```
@@ -14,7 +14,7 @@
   },
 
   "ConnectionStrings": {
-    "Postgres": "Host=192.168.1.20;Port=5438;Username=postgres;Password=!qqG9zfE^Ze!4k;Database=postgres;"
+    "Postgres": "Host=localhost;Port=5438;Username=postgres;Password=!qqG9zfE^Ze!4k;Database=postgres;"
```

Podmianka z powrotem na IP kubernetesowe
```suggestion
    "Postgres": "Host=192.168.1.20;Port=5438;Username=postgres;Password=!qqG9zfE^Ze!4k;Database=postgres;"
```